packages/http/test/routes.test.ts

```diff
@@ -545,4 +551,51 @@ describe("http: routes", () => {
       { verb: "put", path: "/things/{thingId}", params: ["thingId"] },
     ]);
   });
+
+  describe("shared routes", () => {
```

Comment:
@markcowl also left a comment on the original post to make sure that this works for actions, so we should probably add a test that covers that as well.

Reply:
I will add a test for that and some extra validation logic.

Reply:
Added the validation logic and test to verify.